ops/storage-ddls/cassandra-setup.cql

```diff
@@ -79,87 +76,126 @@ CREATE TYPE features (
 );
 
 CREATE TABLE computedtiles (
+    tileid text,
```

Comment:
I wonder if we need to store this given that the tileid is computable from the tilex, tiley and tilez?

```js
tileid = `${zoom}_${row}_${column}`
```